addons/sys_list/initSettings.sqf

```diff
@@ -0,0 +1,122 @@
+// Hint Background Color
+[
+    QGVAR(hintBgColor),
+    "COLOR",
+    localize LSTRING(hintBgColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_BG_BLACK],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Transmit Color
+[
+    QGVAR(transmissionColor),
+    "COLOR",
+    localize LSTRING(transmissionColor_displayName),
+    "ACRE2 UI",
+    [ACRE_NOTIFICATION_YELLOW],
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// Enable PTT color customization
+[
+    QGVAR(showPttColors),
+    "CHECKBOX",
+    localize LSTRING(showPttColors_displayName),
+    "ACRE2 UI",
+    false,
+    false,
+    {}
+] call CBA_Settings_fnc_init;
+
+// PTT1 Color
+[
+    QGVAR(ptt1Color),
+    "COLOR",
+    localize LSTRING(ptt1Color_displayName),
```

Comment:
Description stating that the above setting to enable PTT colors must be enabled wouldn't be bad.